test/ex_twilio/config/env_test.exs

```diff
@@ -0,0 +1,155 @@
+defmodule ExTwilio.Config.EnvTest do
```

Comment:
What do you think about testing`ExTwilio.Config.new/1` and `ExTwilio.Config.from_env/0` directly instead of testing this internal API. That way if someone ever wanted to move the code from `ExTwilio.Config.Env` to somewhere else, the test coverage would on the public API would still remain.

Reply:
I removed `ExTwilio.Config.from_env/0` because it was redundant with `ExTwilio.Config.new/0`. I think mainly `from_env/0` would be used internally in places where the `ExTwilio.Config` module was used with no options available.

I'd like to keep the tests for `ExTwilio.Config.Env`just to keep some reliability on it for the internal usage. I'm planning to add tests for `ExTwilio.Config.new/0` and `ExTwilio.Config.new/1` in later commits.